docs/src/quick_start.rst

```diff
@@ -0,0 +1,214 @@
+
+To check that the installation went well, we can execute a PyMoDAQ’s *control module* called
+:ref:`DAQ_Move <DAQ_Move_module>`, using the
+command
+
+``daq_move``
+
+This prompts the following user interface. In the *Actuator* drop-down list, choose *Mock* and click
+*Initialization*. This will simulate numerically the behavior of a simple actuator, with the reading of its position.
+
+.. figure:: /image/quick_start/mock_actuator.png
+   :width: 300
+
+   The DAQ_Move module.
+
+The basic installation is now complete! :)
+
+Control a real instrument
+-------------------------
+
+In principle, PyMoDAQ can control any instrument.
+However, each specific hardware needs a supplementary package to be compatible with it, which we call an
+:term:`instrument plugin <plugin>`.
+
+List of supported instruments
++++++++++++++++++++++++++++++
+
+Numerous plugins are already available for common scientific equipment suppliers, they are referenced in the
+`list of supported instruments <https://github.com/PyMoDAQ/pymodaq_plugin_manager/blob/main/README.md>`_.
+
+.. figure:: /image/quick_start/supported_instruments_list.png
+
+   List of supported instruments.
+
+We can access it from the *Supported instruments* link on the website’s home page.
+
+.. figure:: /image/quick_start/supported_instruments.png
+
+   Link to the list of supported instruments.
+
+Install the software of the supplier
+++++++++++++++++++++++++++++++++++++
+
+To illustrate concretely the procedure, we suppose that we want to control a *Thorlabs Zelux* camera.
+
+.. figure:: /image/quick_start/zelux_camera.png
+   :width: 200
+
+   A Thorlabs Zelux camera.
+
+This camera is controlled with the
+`ThorCam software <https://www.thorlabs.com/software_pages/ViewSoftwarePage.cfm?Code=ThorCam>`_ that is provided by
+Thorlabs. Let’s download and install it.
+
+.. figure:: /image/quick_start/thorcam.png
+
+   The webpage to download Thorcam.
+
+Once it is installed, connect the camera and check that it is working.
+
+.. warning::
+   It is crucial to first check that your instrument can be controlled with the supplier’s software before trying with
+   PyMoDAQ.
+
+The Plugin Manager
+++++++++++++++++++
+
+Once we have checked that our camera is working, we know that the supplier’s drivers, if any, are installed, and that
+the communication between our camera and our computer is working. It is now time to control it with PyMoDAQ.
+
+We have seen that the *Thorlabs* plugin manages this type of camera.
+
+.. figure:: /image/quick_start/supported_instruments_list_thorlabs.png
+
+   The Zelux camera is supported by the Thorlabs plugin.
+
+In this case, we just have to install the Thorlabs plugin in our environment. To do so, we will use the Plugin Manager
+by executing the following command in our terminal
+
+``plugin_manager``
+
+A window is displayed to easily install the plugin.
+
+.. figure:: /image/quick_start/plugin_manager.png
+   :width: 400
+
+   The Plugin Manager interface.
+
```

Comment:
could you add a note here giving the pip install command for those who do not need the plugin manager?

Reply:
Done.